api-authorization/server/lib/authorization-middleware.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import jwt from 'jsonwebtoken';
+import { ClientError } from './client-error.js';
+
+export async function authorizationMiddleware(req, res, next) {
+  try {
```

Comment:
For synchronous middleware (which should be almost all middleware), exceptions are correctly handled by express, so you don't need the try/catch. It doesn't hurt to have it, it's just not required.